### DISCUSSION TITLE
We need a killer ico for the packages!

### DISCUSSION THREAD

**Discussion by tomlm** [Consolonia/Consolonia#162] [Ideas]
I've been messing around. Do you like any of these?
**Icon1**
![image](https://github.com/user-attachments/assets/a88b557b-39bf-440f-bcfe-cd7365b89443)

**Icon2**
![image](https://github.com/user-attachments/assets/7fbc40e5-52d8-40af-afc2-15704825c55c)

**Icon3**
![image](https://github.com/user-attachments/assets/2dcff1c4-61c2-41e0-a7ae-0a18ccd84b9d)

**Icon4**
Something simple
![image](https://github.com/user-attachments/assets/b8bf8c17-8243-4262-95f6-c9029dfc69aa)

With 4 I was trying to capture a retro command line thing but it just kinda looks meh.

**Comment by tomlm** [comment]
For reference, this is the current avalonia icon
![image](https://github.com/user-attachments/assets/ce6524aa-10c1-4587-8454-4be6e45a3525)

**Comment by jinek** [comment]
I like forth most. But from others first - it's cleanest.

But should not we stick around the TTY topic? Similar to these
![image](https://github.com/user-attachments/assets/deac7b85-712e-4d4e-b7d5-f7bd8d9a9266)
![image](https://github.com/user-attachments/assets/083db5d2-2d55-4914-ae25-26d15873cf06)
![image](https://github.com/user-attachments/assets/4d39e470-1c66-49ab-a662-ebf4caf22116)
![image](https://github.com/user-attachments/assets/9ba2e3d2-829f-4b79-b20c-c9d2df40044d)

**Reply by tomlm** [reply]
I don't mind this one
![image](https://github.com/user-attachments/assets/e27f7367-e146-41eb-b0e4-c98bb82d5d8f)

**Reply by tomlm** [reply]
What I liked about Number 1 from my post because is that it's avalonia's icon with the a turned into a c

**Reply by jinek** [reply]
ok, lets go for first one. It's also recommended by Copilot:

> Recommendation:
    * Icon1 seems to have a slight edge due to its connection to Avalonia, making it recognizable while still representing Consolonia.
    * Based on the feedback, Icon1 is preferred for having a design that aligns with the existing Avalonia icon, which can be beneficial for brand consistency.

**Reply by tomlm** [reply]
What if we make Icon1 Green to make it seem like a CRT terminal
![image](https://github.com/user-attachments/assets/0b496814-19a5-43bd-be5a-99918ec6853a)

**Reply by tomlm** [reply]
![image](https://github.com/user-attachments/assets/ba6a7660-8fd3-492b-8ac1-fde8a30d314c)

**Reply by jinek** [reply]
Also good. Pick whatever you like.
